swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition=0, age=0):
```

Comment:
Here it looks like you started to work on adding `age` to `Item`, but didn't finish. Having this parameter doesn't affect the rest of the code since it has a default value (which lets it be left off everywhere), but since this was an in-progress feature, we should remove it from the parameters, as this becomes confusing about why this unused parameter is here.